financial-templates-lib/ExpiringMultiPartyClient.js

```diff
@@ -1,14 +1,13 @@
 const { delay } = require("./delay");
 
-const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
-
 // A thick client for getting information about an ExpiringMultiParty.
 class ExpiringMultiPartyClient {
-  constructor(empAddress) {
+  constructor(abi, web3, empAddress) {
```

Comment:
Just curious, did you figure out the source of this `artifacts`/`web3` weirdness?

Reply:
It looks to be deep within truffle: https://github.com/trufflesuite/truffle/blob/develop/packages/require/require.js#L119
(resolver is the artifacts thing from what I can tell)

So it looks like web3 is never actually global in terms of being on the window object or whatever. Truffle does some weird wrapping of the imported file.